docs/_data/prague-2021-schedule.yaml

```diff
@@ -0,0 +1,96 @@
+writing_day:
+- time: '10:00'
+  duration: '00:30'
+  title: 'Platform opens'
+- duration: '0:15'
+  title: Morning introduction - Introduction to Sprints & Project Introductions
+- duration: '2:15'
+  title: Team up and start working
+- duration: '1:00'
+  title: Lunch Break (60 mins)
+- time: '17:00'
+  title: End of Writing Day
+
+talks_day1:
+- time: '9:00'
+  duration: '1:00'
+  title: Platform opens
+- duration: '0:45'
+  title: Write the Docs Team - Introduction to Write the Docs
+- title: <b>Unconference Starts</b>
+- duration: '0:40'
+  slug: hitchhiker-s-guide-to-documentation-tools-and-processes-lukas-reu-ner
+- duration: '0:10'
+  title: Break (10 mins)
+- duration: '0:40'
+  slug: more-than-words-reviewing-and-updating-your-information-architecture-lana-brindley-she-her
+- duration: '0:5'
+  title: Switch speakers (5 mins)
```

Comment:
I thought we were just going to use "Break" for all of them? Not terribly emotional about it ;)